packages/manager/apps/dedicated/client/app/dedicated/server/install/image/components/config-drive/translations/Messages_fr_FR.json

```
@@ -0,0 +1,13 @@
+{
+  "dedicated_server_install_image_config_drive": "Activation de ConfigDrive",
+  "dedicated_server_install_image_config_drive_eplain": "ConfigDrive est une partition utilisée par cloud-init lors du premier démarrage de votre serveur. Celle-ci est nécessaire afin de déployer la configuration souhaitée. Vous avez le choix d’activer ou non ce service. Pour plus d’informations, <a href=\"{{ href }}\" target=\"_blank\" rel=\"noopener\">cliquez ici pour consulter notre documentation</a>.",
+  "dedicated_server_install_image_config_drive_activated": "Activé",
+  "dedicated_server_install_image_config_drive_deactivated": "Désactivé",
+  "dedicated_server_install_image_config_drive_hostname": "Hostname du serveur",
+  "dedicated_server_install_image_config_drive_ssh": "Clé SSH (optionnel)",
```

```suggestion
  "dedicated_server_install_image_config_drive_ssh": "Clé SSH (optionnelle)",
```
I'm not sure though if we're talking about the field or the key

---

Mmmmh. I think the field is optional. I don't know :laughing: